app/models/rss_channel.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class RssChannel < ApplicationRecord
+  before_validation :validate_and_set_title
+
+  validates :title, :url, presence: true
```

Comment:
```suggestion
  validates :title, :url, presence: true
  validates :url, unique: true
```

すいません、レビューもれていましたが、上記も追加して欲しいです 🙏